docs/source/tutorials/clean_learning/tabular.ipynb

```diff
@@ -15,16 +15,17 @@
     "Consider Using Datalab\n",
     "<br/>\n",
     "\n",
-    "If interested in detecting a wide variety of issues in your tabular data, check out the [Datalab tabular tutorial](https://docs.cleanlab.ai/stable/tutorials/datalab/tabular.html).\n",
+    "If interested in detecting a wide variety of issues in your tabular data, check out the [Datalab tabular tutorial](https://docs.cleanlab.ai/stable/tutorials/datalab/tabular.html). Datalab can detect many other types of issues other than label issues!\n",
     "</div>"
    ]
   },
   {
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this 5-minute quickstart tutorial, we use cleanlab with scikit-learn models to find potential label errors in a classification dataset with tabular features (numeric/categorical columns). Tabular (or *structured*) data are typically organized in a row/column format and stored in a SQL database or file types like: CSV, Excel, or Parquet. Here we consider a Student Grades dataset, which contains over 900 individuals who have three exam grades and some optional notes, each being assigned a letter grade (their class label). cleanlab automatically identifies _hundreds_ of examples in this dataset that were mislabeled with the incorrect final grade (data entry mistakes). This tutorial shows how to handle noisy labels and produce more robust classification models for your own tabular datasets.\n",
+    "In this 5-minute quickstart tutorial, we use cleanlab with scikit-learn models to find potential label errors in a classification dataset with tabular features (numeric/categorical columns). Tabular (or *structured*) data are typically organized in a row/column format and stored in a SQL database or file types like: CSV, Excel, or Parquet. Here we consider a Student Grades dataset, which contains over 900 individuals who have three exam grades and some optional notes, each being assigned a letter grade (their class label). cleanlab automatically identifies _hundreds_ of examples in this dataset that were mislabeled with the incorrect final grade (data entry mistakes). This tutorial shows how to handle noisy labels and produce more robust classification models for your own tabular datasets. The `CleanLearning` class allows us to detect and filter out such bad data automatically, in order to train a more robust version of any ML model without having to change your existing modeling code! \n",
     "\n",
+    "`CleanLearning` is a handy way to handle noisy labels in ML. \n",
```

Comment:
```suggestion
    "In this 5-minute quickstart tutorial, we use cleanlab with scikit-learn models to find potential label errors in a classification dataset with tabular features (numeric/categorical columns). Tabular (or *structured*) data are typically organized in a row/column format and stored in a SQL database or file types like: CSV, Excel, or Parquet. Here we consider a Student Grades dataset, which contains over 900 individuals who have three exam grades and some optional notes, each being assigned a letter grade (their class label). cleanlab automatically identifies _hundreds_ of examples in this dataset that were mislabeled with the incorrect final grade (data entry mistakes). This tutorial shows how to handle noisy labels and produce more robust classification models for your own tabular datasets. cleanlab's `CleanLearning` class automatically detects and filters out such badly labeled data, in order to train a more robust version of any Machine Learning model. No change your existing modeling code required! \n",
    "\n",
```

Reply:
can we also copy this suggestion to the text.ipynb below?

Reply:
@mturk24 It doesn't look like you addressed my comment? You only added the suggestion in tabular.ipynb but didn't do it in text.ipynb as I requested

Reply:
please be sure not to mark things resolved that aren't resolved

Reply:
Ok addressed this in the newest commit now (with a slight rewording added at the end)